CMakeLists.txt

```diff
@@ -126,7 +126,7 @@ ENDIF()
 #==================================Find Qt5====================================
 
 # The minimum QT5 version we need. Keep as low as necessary.
-SET(QT5_MIN_VERSION 5.3)
+SET(QT5_MIN_VERSION 5.1)
```

Comment:
That was an accident. Didn't mean to commit it. Anyway you can just ignore that one?